src/Libraries/CoreNodeModels/ColorPalette.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Xml;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Media;
+using ProtoCore.AST.AssociativeAST;
+using DSColor = DSCore.Color;
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph;
+
+using CoreNodeModels.Properties;
+
+namespace ColorNode
```

Comment:
I checked bool and integer selector and they bother reference CoreNodeModels.Properties to access the resources.